central/administration/events/datastore/datastore.go

```diff
@@ -39,8 +39,8 @@ func newDataStore(searcher search.Searcher, storage store.Store, writer writer.W
 
 // GetTestPostgresDataStore provides a datastore connected to postgres for testing purposes.
 func GetTestPostgresDataStore(_ testing.TB, pool postgres.DB) DataStore {
```

Comment:
Thanks for starting the new pattern, I think I have come across a few instances. I do not have a strong opinion on it although it improves the readability. Perhaps, it can be addressed in a separate PR if someone has cycles.